packages/accordions/src/styled/StyledIcon.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import {
+  getColor,
+  getLineHeight,
+  retrieveComponentStyles,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'steppers.icon';
+
+interface IStyledIcon {
+  isActive?: boolean;
+  isHorizontal?: boolean;
+}
+
+export const StyledIconFlexContainer = styled.div`
+  display: flex;
+  flex-basis: 100%;
+  justify-content: center;
+  width: 100%;
+`;
+
+export const StyledIcon = styled.div.attrs<IStyledIcon>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledIcon>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  /* stylelint-disable-next-line property-no-unknown */
+  margin-${props => (props.theme.rtl ? 'left' : 'right')}: ${props =>
+  !props.isHorizontal && `${props.theme.space.base * 3}px`};
+  margin-bottom: ${props => props.isHorizontal && props.theme.space.base}px;
+  border-radius: 100%;
+  background: ${props =>
+    props.isActive
+      ? getColor('neutralHue', 600, props.theme)
+      : getColor('neutralHue', 200, props.theme)};
+  width: ${props => props.theme.space.base * 6}px;
+  height: ${props => props.theme.space.base * 6}px;
+  line-height: ${props => getLineHeight(props.theme.lineHeights.lg, props.theme.fontSizes.sm)};
```

Comment:
Maybe a min-width in here to stop this formatting? 
![image](https://user-images.githubusercontent.com/29072694/76112621-5f317a80-5f97-11ea-81f4-08a831434bb1.png)

Reply:
Spoke to @rossmoody, leaving as is because only browser dev tools can force a width that results in this UI.